zubhub_frontend/zubhub/src/hooks/useForceUpdate.hook.js

```diff
@@ -0,0 +1,9 @@
+import { useEffect, useState } from 'react'
+
+export function useForceUpdate () {
```

Comment:
So, I think this type of fix will cause a lot of redraws when trying to attempt this fix. I don't feel that comfortable with this as of now. Is there any other creative way to fix this problem? How is it being handled in Google's form inputs?

Reply:
I am doing research on this, I will keep you posted 👍🏻

Reply:
Hey, as it turns out, the `labelWidth` prop that we were using was removed from MUI OutlinedInput. The new `label` prop calculates the width automatically. So I updated all OutlinedInputs to use the `label` prop.

Here is the link for this change:
https://mui.com/material-ui/migration/v5-component-changes/#outlinedinput

Can you review it again?

Reply:
Nice 🙌